Sources/JsonModel/SerializationFactory.swift

```diff
@@ -200,18 +200,18 @@ open class SerializationFactory : FactoryRegistration {
     
     /// `DateFormatter` to use for coding date-only strings. Default = `_ISO8601DateOnlyFormatter`.
```

Comment:
should the leading underscores on the defaults in these doc comments be removed?